src/bson.rs

```diff
@@ -861,3 +871,9 @@ pub struct RegExp {
     /// unicode.
     pub options: String,
 }
+
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
This name is pretty verbose, but I don't mind that as a subtle way to discourage using this, since it's not really something that is needed